cuda/preconditioner/jacobi_kernels.cu

```diff
@@ -58,7 +58,7 @@ namespace cuda {
  * A compile-time list of block sizes for which dedicated generate and apply
  * kernels should be compiled.
  */
-using compiled_kernels = syn::value_list<int, 1, 13, 16, 32>;
+using compiled_kernels = syn::value_list<int>;
```

Comment:
Do not include this in the commit, it should only exist on your local repository without ever being committed.